docs/installation.rst

```diff
@@ -0,0 +1,85 @@
+Installation
+============
+
+Prerequisites
+-------------
+
+* Python 2.7.x
+
+* pip_ > 7.x
+
+* Toil 3.3.0 or later
+
+.. _pip: https://pip.readthedocs.org/en/latest/installing.html
+
+Basic installation
+------------------
+
+To set up a basic toil-lib installation use
+
+::
+
+    pip install toil-lib
+
+
+Building & testing
+------------------
+
+For developers and people interested in building the project from source the
+following explains how to setup virtualenv to create an environment to use toil-lib
+in.
+
+After cloning the source and ``cd``-ing into the project root, create a
+virtualenv and activate it::
+
+    virtualenv venv
+    . venv/bin/activate
+
+Simply running
+
+::
+
+   make
+
+from the project root will print a description of the available Makefile
+targets.
+
+Once you created and activated the virtualenv, the first step is to install the
+build requirements. These are additional packages that toil-lib needs to be tested
+and built, but not run::
+
+   make prepare
+
+Once the virtualenv has been prepared with the build requirements, running
+
+::
+
+   make develop
```

Comment:
I know my question may be stupid, but I doubt I'll be the only one who gets confused by this.